### ISSUE TITLE
https://cloud.rocket.chat/ - Login Error - "You should be logged to access this page"

### ISSUE THREAD

**Issue by hans4rc** [RocketChat/Rocket.Chat#24919]
For a few weeks now, I can no longer log in to the cloud with my account. No matter if with email link or with username and password, I get the same message.

My account exists since December 2020.

https://cloud.rocket.chat/

![image](https://user-images.githubusercontent.com/66670106/159722981-c3e47334-2fb2-4d6c-a24d-c4c24d82b3e7.png)

**Comment by dudanogueira** [comment]
Please, provide the information asked, such as version, deployment method, etc.

**Comment by dudanogueira** [comment]
Hi! When you receive the link for login and click on it, what happens exactly?

It just appears this message on a blank page?

**Comment by hans4rc** [comment]
Yes exactly, the message is displayed on a white page. After a few seconds the message disappears and a white page remains.

I have tried the login on FireFox, Opera, Google Chrome and Edge. And I have cleared the cache. It is always the same. White page.

**Comment by dudanogueira** [comment]
@hans4rc would you mind emailing me the username associated with that account?
my email: duda.nogueira@rocket.chat
thanks!

**Comment by JKawohl** [comment]
@dudanogueira unfortunately i am running into the exact same error. Did you guys find a solution yet?

**Comment by hans4rc** [comment]
Unfortunately no, I still cannot log in.

**Comment by katzecase** [comment]
Same problem here, and if go to the reset password page, enter anything with @gmail.com would shows "Please use a work email" message. Same with @protonmail.com .

**Comment by daurpam** [comment]
Another one with the same issue, can't login even using password, sending link or whatever. Impossible to install apps from marketplace without login to cloud.rocket.chat

**Comment by manfredsteger** [comment]
same problem here - i tried to make a new account and connect it with my rc instance and get a new error:
<img width="300" alt="rc-cloud-login-error" src="https://user-images.githubusercontent.com/8418059/172842108-d35d189a-a6cd-4458-98aa-ab888d9fc88c.png">



After the redirect to the cloud login page my rc instance seems to be connected, but if i try to download an app from the marketplace -> please login to ...

**Comment by manfredsteger** [comment]
here is the coding solution: https://github.com/RocketChat/Rocket.Chat/issues/17535#issuecomment-636024380
